1.1/validator.ttl

```diff
@@ -0,0 +1,154 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@base <http://www.opengis.net/def/geosparql/validator/rule/> .
+
+
+<http://www.opengis.net/def/geosparql/validator>
+	a owl:Ontology ;
+  	dcterms:title "GeoSPARQL RDF Shapes Validator"@en ;
+	dcterms:description """This is a [Shapes Constraint Language (SHACL)](https://www.w3.org/TR/shacl/) file that specifies rules for RDF data. It can be used to test the conformance of data to the GeoSPARQL standard.
+	
+As of GeoSPARQL 1.1, this validator is not normaltive, only informative, however this is likely to be the basis of future, normative, validators."""@en ;
```

Comment:
"normative" (typo)

Reply:
Fixed in Commit https://github.com/opengeospatial/ogc-geosparql/pull/152/commits/747ee942174a35a8f3e2d676b0b4671d4c86fd7f